yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -108,7 +108,7 @@ describe('AVM simulator: transpiled Noir contracts', () => {
       const bytecode = getAvmTestContractBytecode('u128_addition_overflow');
       const results = await new AvmSimulator(initContext()).executeBytecode(bytecode);
       expect(results.reverted).toBe(true);
-      expect(results.revertReason?.message).toEqual('Reverted with output: attempt to add with overflow');
+      expect(results.revertReason?.message).toEqual('Assertion failed: attempt to add with overflow');
```

Comment:
For whatever reason I was getting this error on master:
<img width="874" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/c255dcf6-6ae6-4af7-be87-b8a3d2649929">

Not sure, why it was not caught by CI.